plugins/usbdmx/UsbDmxPlugin.h

```diff
@@ -59,6 +61,9 @@ class UsbDmxPlugin: public ola::Plugin {
   ola_plugin_id Id() const { return OLA_PLUGIN_USBDMX; }
   std::string PluginPrefix() const { return PLUGIN_PREFIX; }
 
+  void ConflictsWith(
+    std::set<ola_plugin_id>* conflicting_plugins) const override;
```

Comment:
Override is C++11, which we're not exclusively targeting yet.

Reply:
Time to switch? Removed `override` :-(.

Reply:
Probably, see some initial discussion here: https://github.com/OpenLightingProject/ola/issues/1028#issuecomment-301513614

And my general thoughts here:
https://github.com/OpenLightingProject/ola/issues/1028#issuecomment-493076260